app/javascript/dashboard/i18n/locale/en/settings.json

```
@@ -4,7 +4,7 @@
     "TITLE": "Profile Settings",
     "BTN_TEXT": "Update Profile",
     "DELETE_AVATAR": "Delete Avatar",
-    "AVATAR_DELETE_SUCCESS": "Avatar has been deleted successfully",
+    "AVATAR_DELETE_SUCCESS": "Avatar has been deleted successfully, please click on update profile to save the changes",
```

This change is unnecessary because the delete function automatically removes it from the profile.

```suggestion
    "AVATAR_DELETE_SUCCESS": "Avatar has been deleted successfully",
```